clients/include/testing_geam_strided_batched.hpp

```diff
@@ -0,0 +1,593 @@
+                                                          batch_count),
+                          rocblas_status_invalid_pointer);
+
+    EXPECT_ROCBLAS_STATUS(rocblas_geam_strided_batched<T>(handle,
+                                                          transA,
+                                                          transB,
+                                                          M,
+                                                          N,
+                                                          &h_alpha,
+                                                          dA,
+                                                          lda,
+                                                          stride_a,
+                                                          nullptr,
+                                                          dB,
+                                                          ldb,
+                                                          stride_b,
+                                                          dC,
+                                                          ldc,
+                                                          stride_c,
+                                                          batch_count),
+                          rocblas_status_invalid_pointer);
+
+    EXPECT_ROCBLAS_STATUS(rocblas_geam_strided_batched<T>(nullptr,
+                                                          transA,
+                                                          transB,
+                                                          M,
+                                                          N,
+                                                          &h_alpha,
+                                                          dA,
+                                                          lda,
+                                                          stride_a,
+                                                          &h_beta,
+                                                          dB,
+                                                          ldb,
+                                                          stride_b,
+                                                          dC,
+                                                          ldc,
+                                                          stride_c,
+                                                          batch_count),
+                          rocblas_status_invalid_handle);
+}
+
+template <typename T>
+void testing_geam_strided_batched(const Arguments& arg)
+{
+    rocblas_operation transA = char2rocblas_operation(arg.transA);
+    rocblas_operation transB = char2rocblas_operation(arg.transB);
+
+    rocblas_int M = arg.M;
+    rocblas_int N = arg.N;
+
+    rocblas_int lda         = arg.lda;
+    rocblas_int ldb         = arg.ldb;
+    rocblas_int ldc         = arg.ldc;
+    rocblas_int stride_a    = arg.stride_a;
```

Comment:
rocblas_stride type.   These will save you from the missing size promotions later, but you can use them as the first calculation so the later additions won't overflow.